ir/src/main/scala/org/scalajs/core/ir/Infos.scala

```diff
@@ -356,6 +359,9 @@ object Infos {
         case ClassOf(cls) =>
           builder.addAccessedClassData(cls)
 
+        case JSLoadConstructor(cls) =>
+          builder.addInstantiatedClass(cls.className)
+
```

Comment:
Hmm... This is actually an over approximation... If the `JSLoadConstructor` is inside an `isInstanceOf`, it does not make the class instantiated...